components/login/index.jsx

```diff
@@ -78,28 +104,24 @@ export default function Login() {
         alt="Inrupt PodBrowser"
         className={bem("login-form__logo")}
       />
-      <LoginButton
```

Comment:
I've explicitly opted to not use the `LoginButton` component, as we actually have additional logic that we need to call before the login redirect happens (specifically the `persist()` call) — in the future, the SDK may have a `beforeRedirect` event, that would allow us to install this handler at a global level, rather than doing it like it's done here.